sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePlugin.scala

```diff
@@ -12,6 +10,7 @@ object ScalaNativePlugin extends AutoPlugin {
 
     val ScalaNativeCrossVersion = sbtplugin.ScalaNativeCrossVersion
 
+    val nativeOrgName = "org.scala-native"
```

Comment:
There's no need to make it public. All vals defined here would be exposed to sbt plugin users (and typically autoimported)